Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -256,6 +254,28 @@ theorem le_measure_diff : μ s₁ - μ s₂ ≤ μ (s₁ \ s₂) :=
 
 #align measure_theory.le_measure_diff MeasureTheory.le_measure_diff
 
+/-- If the measure of the symmetric difference of two null measurable sets is finite,
+then one has infinite measure if and only if the other does. -/
+theorem measure_eq_top_iff_of_symmDiff (hs : NullMeasurableSet s μ) (ht : NullMeasurableSet t μ)
+    (hμst : μ (s ∆ t) ≠ ∞) : μ s = ∞ ↔ μ t = ∞ := by
```

Comment:
I'm wondering whether you can get rid of the measurability assumption by using `measure_eq_iInf`/`measure_eq_iInf'`. Would you be willing to try?

Reply:
Ok I'll try.